pykusto/functions.py

```diff
@@ -571,7 +572,7 @@ def pack_dictionary() -> MappingExpression:
         """
         https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/packdictionaryfunction
         """
-        raise NotImplemented  # pragma: no cover
+        return MappingExpression("pack_dictionary()")
```

Comment:
pack_dictionary is an alias for pack and it has arguments

Reply:
yes, it doesn't matter which one I use, so I thought it's best to keep the dual names, since Kusto allows it